eldritch/encounters.py

```diff
@@ -367,6 +367,136 @@ def Isle7(char):
   clues = events.Gain(char, {"clues": 2})
   return events.PassFail(char, check, clues, events.Nothing())
 
+def Hospital1(char):
+  return events.Loss(char, {"clues": 1})
+def Hospital2(char):
+  prep = events.CombatChoice(char, "Choose weapons to fight the corpse")
+  check = events.Check(char, "combat", -1)
+  won = events.Gain(char, {"clues", 1})
+  lost = events.ForceMovement(char, "Uptown")
+  cond = events.Conditional(char, check, "successes", {0: lost, 1: won})
+  return events.Sequence([prep, check, cond], char)
+def Hospital3(char):
+  die = events.DiceRoll(char, 1)
+  # TODO: On a 1-3 gain that many stamina, on a 4-6 nothing happens.
+  return events.Nothing(char)
+def Hospital4(char):
+  check = events.Check(char, "luck", -1)
+  gain = events.Gain(char, {"sanity": 2, "dollars": 3})
+  fail1 = events.Lose(char, {"sanity": 2})
+  fail2 = events.ForceMove(char, "Uptown")
+  fail = events.Sequence([fail1, fail2], char)
+  return events.PassFail(char, check, gain, fail)
+def Hospital5(char):
+  check = events.Check(char, "sneak", -1)
+  gain = events.Draw(char, "spells", 1)
+  return events.PassFail(char, check, gain, events.Nothing(char))
+def Hospital6(char):
+  check = events.Check(char, "will", -1)
+  item = events.Draw(char, "unique", 1)
+  fail1 = events.Lose(char, {"sanity":1})
+  fail2 = events.ForceMovement(char, "Uptown")
+  fail = events.Sequence([fail1, fail2], char)
+  return events.PassFail(char, check, item, fail)
+def Hospital7(char):
+  check = events.Check(char, "lore", 0)
+  clue = events.Gain(char, {"clues": 1})
+  return events.PassFail(char, check, clue, events.Nothing(char))
+
+def Woods1(char):
+  box = events.Check(char, "luck", 0)
+  foot = events.Lose(char, {"sanity": 1})
+  common = events.Draw(char, "common", 1)
+  unique = events.Draw(char, "unique", 1)
+  jewelry = events.Gain(char, {"dollars": 1})
+  cond = events.Conditional(char, box, "successes", {0: foot, 1: common, 2: unique, 3: jewelry})
+  open_box = events.Sequence([box, cond], char)
+  return events.BinaryChoice(char, "Open the locked box?", "Yes", "No", open_box, events.Nothing(char))
+def Woods2(char):
+  check = events.Check(char, "sneak", -1)
+  make_check = events.PassFail(char, check, events.Nothing, events.Loss(char, {"stamina":2}))
+  leave = events.ForceMovement(char, "Uptown")
+  return events.Sequence([make_check, leave], char)
+def Woods3(char):
+  check = events.Check(char, "sneak", -2)
+  shotgun = events.DrawSpecific(char, "common", "Shotgun")
+  fail = events.Sequence([events.Loss(char, {'stamina': 2}), events.ForceMovement(char, "Uptown")], char)
+  return events.PassFail(char, check, shotgun, fail)
+def Woods4(char):
+  check = events.Check(char, "luck", -1)
+  bushwhack1a = events.ItemChoice(char, "Choose first item to lose")
```

Comment:
Note that you'll also need a prerequisite on the character having enough items to lose.